tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 # TODO: re-enable py33,py34 when bravado-core is py3x compatible - https://github.com/Yelp/bravado-core/issues/10
-envlist = py26,py27,py33,py34,pep8
+envlist = py26,py27,py33,py34,py35,{py27,py35}-pyramid15,pep8
```

Comment:
Correct!  Since `deps` runs before the `installcommand` it'll already have pyramid installed when the tests run and wont pull it in:

```
$ .tox/py35-pyramid15/bin/pip freeze|grep pyramid
pyramid==1.5.4
pyramid-swagger==2.2.2


$ .tox/py35/bin/pip freeze|grep pyramid
pyramid==1.6.1
pyramid-swagger==2.2.2
```